typescript/web/src/pages/[workspaceSlug]/datasets/index.tsx

```diff
@@ -12,12 +12,19 @@ import { WorkspaceTabBar } from "../../../components/layout/tab-bar/workspace-ta
 import { NavLogo } from "../../../components/logo/nav-logo";
 import { Meta } from "../../../components/meta";
 import { WorkspaceSwitcher } from "../../../components/workspace-switcher";
+import { UpdatePlanBanner } from "../../../components/workspaces/update-plan-banner";
+import { WorkspaceStatus } from "../../../graphql-types";
 import { useWorkspace } from "../../../hooks";
 import { BoolParam, IdParam } from "../../../utils/query-param-bool";
 
 const Body = () => {
-  const { slug: workspaceSlug } = useWorkspace();
+  const { slug: workspaceSlug, status } = useWorkspace();
   const { isReady } = useRouter();
+  const isWorkspaceActive =
+    status === WorkspaceStatus.active ||
+    status === WorkspaceStatus.trialing ||
+    status === WorkspaceStatus.incomplete;
```

Comment:
While we're at introducting a @labelflow/core package, I'd move this logic into a shared `isWorkspaceActive(status: WorkspaceStatus)` function. Or at least in `@labelflow/common-resolvers` if you don't want to follow the _"little-bit-less quick"_ solution I've detailed above.

Reply:
Why not leave it here as it's not yet used anywhere else ?

Reply:
Because we'll need is as-soon as we'll have to block features if a workspace is inactive.